deploy/gke-marketplace-app/client-sample/locustfile_bert_large.py

```diff
@@ -67,4 +67,4 @@ def on_start(self):
             self.data = json.load(f)
 
         self.url1 = '{}/v2/models/{}/infer'.format(self.environment.host,
-                                                   'bert_large')
+                                                   'bert')
```

Comment:
nit: Should the file name change from `locustfile_bert_large.py` to `locustfile_bert.py`?

Reply:
fixed